src/main/java/ladder/LadderApplication.java

```diff
@@ -0,0 +1,17 @@
+package ladder;
+
+import ladder.domain.Ladder;
+import ladder.domain.Players;
+import ladder.view.InputView;
+import ladder.view.ResultView;
+
+public class LadderApplication {
+
+  public static void main(String[] args) {
+    Players players = InputView.playerNames();
```

Comment:
```suggestion
    Players players = InputView.players();
```

Players 객체를 반환하는데, 굳이 playerNames 라고 구분지으신 이유가 있으시나요?
궁금해서 여쭈어봅니다.
저는 메서드명만 보았을 때는 playerNames 라는 또다른 일급 컬렉션이 있다고 생각했습니다.
수정해달라는 게 아니라, 같이 협업하는 동료가 이런식으로 생각할수도 있겠구나 하고 넘어가주시면 좋을 것 같습니다.

Reply:
저는 Player 엔 이름외의 나이 또는 이메일 같은 다른 데이터가 추후에 포함될거라고 생각했습니다.
다음과 같은 확장성을 고려해서 Player에 Name이라는 래퍼 클래스를 정의했는데요.

정환님 말하신것처럼 현재는 Player 와 Name는 굳이 분리할 필요는 없어 보이네요. 
(오버엔지니어링....) 다른 동료가 명료하지?않는 메서드 때문에 혼란을 주는 요소 같네요.😅